doc/command-line.md

```diff
@@ -0,0 +1,138 @@
+
+# Using sourmash from the command line
+
+
+From the command line, sourmash can be used to compute [MinHash
+sketches][0] from DNA
+sequences, compare them to each other, and plot the results.  This
+allows you to estimate sequence similarity quickly and accurately.
+
+Please see the [mash software][1] http://mash.readthedocs.io/en/latest/__
+ and the [mash paper (Ondov et al., 2016)][2]
+for background
+information on how and why MinHash sketches work.
+
+______
+
+sourmash uses a subcommand syntax, so all commands start with
+`sourmash` followed by a subcommand specifying the action to be
+taken.
+
+Contents
+
+* [An example](## An example)
+* [The `sourmash` command and its subcommands](## The sourmash command and its subcommands)
+  * [`sourmash compute`](### `sourmash compute`)
+
+  * [`sourmash compare`](### `sourmash compare`)
+
+  * [`sourmash plot`](### `sourmash plot`)
+
+## An example
+
+
+Grab three bacterial genomes from NCBI:
+```
+   curl -L -O ftp://ftp.ncbi.nlm.nih.gov/genomes/refseq/bacteria/Escherichia_coli/reference/GCF_000005845.2_ASM584v2/GCF_000005845.2_ASM584v2_genomic.fna.gz
+   curl -L -O ftp://ftp.ncbi.nlm.nih.gov/genomes/refseq/bacteria/Salmonella_enterica/reference/GCF_000006945.1_ASM694v1/GCF_000006945.1_ASM694v1_genomic.fna.gz
+   curl -L -O ftp://ftp.ncbi.nlm.nih.gov/genomes/refseq/bacteria/Sphingobacteriaceae_bacterium_DW12/latest_assembly_versions/GCF_000783305.1_ASM78330v1/GCF_000783305.1_ASM78330v1_genomic.fna.gz
```

Comment:
Not a huge deal, but since other changes are required, should remove the leading spaces here.

Reply:
And in the other code blocks in this file.